GetF!t/NavigationManager.swift

```diff
@@ -25,6 +30,15 @@ class NavigationManager: ObservableObject {
         }
     }
 
+    private func nameForHistory(titleText: LocalizedStringKey) -> String {
+        switch titleText {
+        case LocalizedStringProvider.ExercisesNames.squat: return StringProvider.ExercisesNamesVideo.squat
+        case LocalizedStringProvider.ExercisesNames.stepUp: return StringProvider.ExercisesNamesVideo.stepUp
+        case LocalizedStringProvider.ExercisesNames.burpee: return StringProvider.ExercisesNamesVideo.burpee
+        default: return StringProvider.ExercisesNamesVideo.sunSalute
+        }
+    }
```

Comment:
tu by som asi šla cestou, že budeš mať funkciu ktorá ti vráti string pre HistoryViewModel. Teda táto funkcia bude public a budeš ju volať v TimerView. A odstrániš ten published  titleTextForHistoryStore